umh.docs.umh.app/content/en/docs/production-guide/administration/access-services-from-outside-cluster.md

```diff
@@ -73,16 +74,23 @@ LoadBalancer service.
 
 ### Create a LoadBalancer service
 
-If you are looking to expose th Kafka broker, follow the instructions in the
+If you are looking to expose the Kafka broker, follow the instructions in the
 [Access Kafka outside the cluster](/docs/production-guide/administration/access-kafka-outside-cluster/)
 page.
 
 For any other microservice, follow these steps to enable the LoadBalancer service:
 
-1. Open {{< resource type="lens" name="name" >}} and navigate to **Network** >
-    **Services**.
-2. Select the service and click the **Edit** button.
-3. Scroll down to the `status.loadBalancer` section and change it to the following:
+1. Execute the following command to get the list of available services and remember 
+   the service to be exposed.
+   ```bash
+   sudo $(which kubectl) get svc -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
+   ```
+2. To edit the service, run the following command:
```

Comment:
"Start editing the service configuration by running this command:"

Reply:
updated